web/src/lib/components/shared-components/side-bar/side-bar.svelte

```diff
@@ -24,6 +26,14 @@
 		};
 	};
 
+	const getFavoriteCount = async () => {
+		const { data: assets } = await api.assetApi.getAllAssets(true);
+
+		return {
+			favorites: assets.length
+		};
+	};
+
```

Comment:
Similarly, this works for now, but we might want to refactor this in the future so it doesn't have to load, say 1k assets just to show the count.

Reply:
I may try my hand at this. There are a couple examples of counts `/asset/count-by-time-bucket`, `/asset/count-by-user-id`, and `/album/count-by-user-id`. The question I have is would it be better to name a `/favorite/count-by-user-id` endpoint or `/asset/count-favorites`?

Reply:
Or maybe better yet, extend the `/asset/count-by-user-id` to take an optional `isFavorite` parameter.

Reply:
> Or maybe better yet, extend the /asset/count-by-user-id to take an optional isFavorite parameter.

This works